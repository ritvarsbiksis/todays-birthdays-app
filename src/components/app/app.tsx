import { Header } from 'components/header/header'
import { BirthdaysPage } from 'pages/birthdays-page/birthdays-page'

export const App = () => (
  <div style={{ padding: '20px' }}>
    <Header />
    <BirthdaysPage />
  </div>
)
