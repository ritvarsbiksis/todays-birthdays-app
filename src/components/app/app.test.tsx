import { createSerializer } from '@emotion/jest'
import { render, screen } from '@testing-library/react'

import { App } from './app'

expect.addSnapshotSerializer(createSerializer())

describe('App', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })

  test('renders header element', () => {
    render(<App />)
    expect(screen.getByRole('banner')).toBeInTheDocument()
  })
})
