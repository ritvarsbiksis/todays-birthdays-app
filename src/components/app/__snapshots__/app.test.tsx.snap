// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  margin-bottom: 10px;
}

.emotion-1 {
  margin: 0;
  color: darkslategray;
  text-transform: uppercase;
  font-weight: 300;
}

<div
    style="padding: 20px;"
  >
    <header
      class="emotion-0"
    >
      <h1
        class="emotion-1"
      >
        Today's birthdays
      </h1>
    </header>
    <button>
      Load data
    </button>
  </div>
</DocumentFragment>
`;
